### PR TITLE
update to latest upstream main

### PR DESCRIPTION
**Pull request by bavarianbidi**
This PR updates `giantswarm-main` to latest upstream `main`.

### CODE REVIEW

File: controllers/cluster_scripts/cloud_init.tmpl [hunk 1]
```diff
@@ -104,26 +104,7 @@ write_files:
...
-    Environment="http_proxy={{.HTTPProxy}}"
-    Environment="https_proxy={{.HTTPSProxy}}"
-    Environment="no_proxy={{.NoProxy}}"
-    Environment="NO_PROXY={{.NoProxy}}"
-    END
-    systemctl daemon-reload
-    systemctl restart containerd
-    vmtoolsd --cmd "info-set guestinfo.postcustomization.proxy.setting.status successful" {{- if .NvidiaGPU }}
-
+    {{- if .NvidiaGPU }}
```
Comment (erkanerol) [inline controllers/cluster_scripts/cloud_init.tmpl:107]: @bavarianbidi this is our change. right?

**Comment by erkanerol** [comment]
@bavarianbidi do you want us to test this PR ?

File: controllers/cluster_scripts/cloud_init.tmpl (hunk 1 above)
Comment (bavarianbidi) [reply controllers/cluster_scripts/cloud_init.tmpl:107]: Yes, we already running this change for CAPVCD

**Comment by bavarianbidi** [comment]
> @bavarianbidi do you want us to test this PR ?

i don't think it's required to test this as you can see from the commits only the CAPVCD 1.0.0 documentation PR is added to `giantswarm-main` branch.

All other changes (my two changes) are just appended (due interactive rebasing) and are already on `giantswarm-main`.

**Review by erkanerol — APPROVED** [review]

**Pull request merged**